### PR TITLE
Make cut off and drop off names consistent

### PR DESCRIPTION
**Pull request by greschd**
Rename the "cut off" and "drop off" related features to make them consistent:
- `Cutoff` -> `CutOff`
- `cutoff` -> `cut_off`
- `Dropoff` -> `DropOff`
- `dropoff` -> `drop_off`

Partly addresses #637.

### CODE REVIEW

File: doc/source/user_guide/howto/print_model.rst [hunk 1]
```diff
@@ -90,7 +90,7 @@ The ``hide_empty`` label can be set to ``False`` to also show empty groups:
         Cylindrical Selection Rules
         Spherical Selection Rules
         Tube Selection Rules
-        Cutoff Selection Rules
+        CutOff Selection Rules
```
Comment (greschd) [inline doc/source/user_guide/howto/print_model.rst:93]:
```suggestion
        Cut Off Selection Rules
```

File: src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py [hunk 2]
```diff
@@ -63,50 +63,50 @@
...
+    "CutOffSelectionRuleElementalData",
+    "CutOffSelectionRuleNodalData",
 ]
 
 
 @dataclasses.dataclass
-class CutoffSelectionRuleElementalData(ElementalData):
-    """Represents elemental data for a Cutoff Selection Rule."""
+class CutOffSelectionRuleElementalData(ElementalData):
+    """Represents elemental data for a CutOff Selection Rule."""
```
Comment (greschd) [inline src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py:74]:
```suggestion
    """Represents elemental data for a Cut Off Selection Rule."""
```

File: src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py [hunk 3]
```diff
@@ -63,50 +63,50 @@
...
 
     normal: VectorData | None = None
 
 
 @dataclasses.dataclass
-class CutoffSelectionRuleNodalData(NodalData):
-    """Represents nodal data for a Cutoff Selection Rule."""
+class CutOffSelectionRuleNodalData(NodalData):
+    """Represents nodal data for a CutOff Selection Rule."""
```
Comment (greschd) [inline src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py:81]:
```suggestion
    """Represents nodal data for a Cut Off Selection Rule."""
```

File: src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py [hunk 4]
```diff
@@ -63,50 +63,50 @@
...
+class CutOffSelectionRuleNodalData(NodalData):
+    """Represents nodal data for a Cut Off Selection Rule."""
 
 
 @mark_grpc_properties
 @register
-class CutoffSelectionRule(CreatableTreeObject, IdTreeObject):
-    """Instantiate a Cutoff Selection Rule.
+class CutOffSelectionRule(CreatableTreeObject, IdTreeObject):
+    """Instantiate a Cut Off Selection Rule.
```
Comment (roosre) [inline src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py:87]: I would use Cut-Off and Drop-Off in doc strings to be consistent with the ACP documentation.
```suggestion
    """Instantiate a Cut-Off Selection Rule.
```

File: examples/modeling_features/03-advanced-selection-rules.py [hunk 5]
```diff
@@ -179,37 +179,37 @@
 plotter.show()
 
 # %%
-# Create a cutoff selection rule
-# ------------------------------
+# Create a cut off selection rule
```
Comment (roosre) [inline examples/modeling_features/03-advanced-selection-rules.py:182]:
```suggestion
# Create a cut-off selection rule
```

File: src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py [hunk 6]
```diff
@@ -63,50 +63,50 @@
...
+    "CutOffSelectionRuleElementalData",
+    "CutOffSelectionRuleNodalData",
 ]
 
 
 @dataclasses.dataclass
-class CutoffSelectionRuleElementalData(ElementalData):
-    """Represents elemental data for a Cutoff Selection Rule."""
+class CutOffSelectionRuleElementalData(ElementalData):
+    """Represents elemental data for a Cut Off Selection Rule."""
```
Comment (roosre) [inline src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py:74]:
```suggestion
    """Represents elemental data for a Cut-Off Selection Rule."""
```

File: src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py [hunk 7]
```diff
@@ -63,50 +63,50 @@
...
 
     normal: VectorData | None = None
 
 
 @dataclasses.dataclass
-class CutoffSelectionRuleNodalData(NodalData):
-    """Represents nodal data for a Cutoff Selection Rule."""
+class CutOffSelectionRuleNodalData(NodalData):
+    """Represents nodal data for a Cut Off Selection Rule."""
```
Comment (roosre) [inline src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py:81]:
```suggestion
    """Represents nodal data for a Cut-Off Selection Rule."""
```

File: src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py [hunk 8]
```diff
@@ -63,50 +63,50 @@
...
-    """Instantiate a Cutoff Selection Rule.
+class CutOffSelectionRule(CreatableTreeObject, IdTreeObject):
+    """Instantiate a Cut Off Selection Rule.
 
     Parameters
     ----------
     name :
-        Name of the Cutoff Selection Rule.
-    cutoff_rule_type :
+        Name of the Cut Off Selection Rule.
```
Comment (roosre) [inline src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py:92]:
```suggestion
        Name of the Cut-Off Selection Rule.
```

File: tests/unittests/test_tree_printer.py [hunk 9]
```diff
@@ -116,8 +116,8 @@ def case_more_objects(acp_instance, model_data_dir):
                 'CylindricalSelectionrule'
             Tube Selection Rules
                 'TubeSelectionrule'
-            Cutoff Selection Rules
-                'CutoffSelectionrule'
+            Cut Off Selection Rules
```
Comment (roosre) [inline tests/unittests/test_tree_printer.py:119]: Cut Off is fine in the print_model.

**Review by roosre — APPROVED** [review]
Did you also check if the code is consistent w.r.t. SnapTo and snap_to? Or is snapto used somewhere?

**Review by roosre — APPROVED** [review]

**Comment by greschd** [comment]
> Did you also check if the code is consistent w.r.t. SnapTo and snap_to? Or is snapto used somewhere?

That's a good point, I had forgotten about SnapTo. I now checked, and it appears to be consistent already.

**Pull request merged**